data/json/items/comestibles/seed.json

```diff
@@ -14,6 +14,23 @@
         "symbol": ".",
         "color": "brown"
     },
+    {
+        "id": "seed_bee_balm",
+        "copy-from": "seed",
+        "type": "COMESTIBLE",
+        "name": "bee balm seeds",
+        "name_plural": "bee balm seeds",
+        "description": "Some bee balm seeds.  You could probably plant these.",
+        "charges": 2,
+        "material": "veggy",
```

Comment:
`material: fruit` should **not** be a property of `abstract: seed` as there are many overrides. The same applies to `charges`. Seeds should either all be the same `color` (I'd prefer this although others may disagree) otherwise again this shouldn't be part of the abstract.